CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Release Notes for Stripe
 
+## Unreleased
+
+- Fixed a styling issue with the element's meta card footer.
```

Comment:
Probably good to specify which element's meta card

Reply:
Done